src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WCUTBACK

```diff
@@ -16,6 +16,10 @@
       "name": "GOR_LIMIT",
       "value_type": "DOUBLE"
     },
+    {
+      "name": "GLR_LIMIT",
+      "value_type": "DOUBLE"
+    },
```

Comment:
This probably needs a `dimension` field too.  The proposed specification is okay for METRIC, LAB, and PVT-M, but will be incorrect for the case of FIELD unit conventions.  Somewhat misleadingly, adding
```json
"dimension": "GasDissolutionRatio"
```
here will do the right thing.

If you do add that, it would be prudent to add the same for the `GOR_LIMIT` too.

Reply:
Maybe also for `WGR_LIMIT`? (with `GasDissolutionRatio`?

Reply:
> Maybe also for `WGR_LIMIT`?

Yes, absolutely.

> with `GasDissolutionRatio`?

Not quite.  In that case it will be the even more obtuse `OilDissolutionRatio` (which mostly exists for the `RV` quantity).

Reply:
Sorry&mdash;I misspoke.  It should be GasDissolution**Factor** and OilDissolution**Factor**, not **Ratio**.  Otherwise it should be okay.

Reply:
Similarly for the `*_LIMIT_REVERSE` items too.